### PR TITLE
Support Intellij 2023

### PR DESCRIPTION
**Pull request by julioromano**
On the flip side minimum version is now 2022.3.
I really hope this works, I tested it by running `runIde` and it highlights .udl files. Hope this is enough. I'm an intellij plugin noob.

Fixes https://github.com/Lonami/uniffi-dl/issues/3

### CODE REVIEW

**Comment by Lonami** [comment]
> I'm an intellij plugin noob

Now we have something in common!

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+networkTimeout=10000
```
Comment (Lonami) [inline gradle/wrapper/gradle-wrapper.properties:4]: Unsure about the changes to the gradle files. How come they happened? I would prefer if they were in a separate commit.

Comment (julioromano) [reply gradle/wrapper/gradle-wrapper.properties:4]: It's part of the gradle wrapper upgrade to v8.1.1 (latest) as was suggested in the IntelliJ plugin dev docs (to always use latest cradle version).

Comment (julioromano) [reply gradle/wrapper/gradle-wrapper.properties:4]: I meant that I didn't change those files manually. it was the gradle wrapper upgrade script that did it.

Comment (julioromano) [reply gradle/wrapper/gradle-wrapper.properties:4]: Here's some background: https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

**Pull request merged**